### ISSUE TITLE
Custom Paths Automatically Appended with Other Links After Plugin Upgrade to 3.37.1

### ISSUE THREAD

**Issue by rosejeck** [chathub-dev/chathub#1266]
**Describe the bug**

![image](https://github.com/user-attachments/assets/e1315db2-f1fd-4267-ae4d-dae10b7ad59e)

**Desktop (please complete the following information):**
 - OS: [e.g. Windows]
 - Browser [e.g. chrome, brave]
 - ChatHub Version [you can find it in ChatHub Settings]

**Comment by wong2** [comment]
That's right, can you give a more specific example of a problem it caused? Like a provider you're using that doesn't support the /v1 format of the API?

**Comment by rosejeck** [comment]
The custom interface I've been using worked fine in previous versions, but now after configuration, it automatically appends /v1/chat/completions to the end.

**Comment by wong2** [comment]
Did you build the custom interface yourself? Or are you using a hosted or open-source one?

**Comment by rosejeck** [comment]
YES,Built your own custom interface

**Comment by wong2** [comment]
Can you update the implementation? Because /chat/completions is actually required to be OpenAI compatible

**Comment by rosejeck** [comment]
No, because their own encapsulation of the call interface with parameters, the original plug-in supports custom configuration, now how to change does not support

**Comment by rosejeck** [comment]
![企业微信截图_17237723824310](https://github.com/user-attachments/assets/02e4d320-c2b4-4429-af4a-52ded496d79a)
It is also a custom interface, why should I add something to the interface I configured

**Comment by wong2** [comment]
Hi, you can speak Chinese here

**Comment by rosejeck** [comment]
![image](https://github.com/user-attachments/assets/a364b3fd-84e7-4316-a6e6-3e6533422329)
After the configuration, save, to add /v1, I am a custom configuration, should match what to perform what

**Comment by wong2** [comment]
我想我明白你的意思。
做出这个改动的原因是，很多用户不会设置完整的api路径，所以我们进行了简化，只需要输入api host部分即可。
同时我们也在收集像你遇到的情况这样的bad case，以便进行兼容，所以我需要知道你的自定义服务所使用的接口是什么样的

**Comment by rosejeck** [comment]
![image](https://github.com/user-attachments/assets/ddc5f25c-b5be-4d27-ac44-d03cbd3871fc)
这样的配置地址，请求的时候直接调用我配的地址就行，不需要再拼接
![image](https://github.com/user-attachments/assets/0fe2d80f-ab8a-4fb1-9d1a-56b0bc21db50)
